src/openforms/forms/api/serializers/form_variable.py

```diff
@@ -21,23 +21,24 @@ def validate(self, attrs):
         static_data_keys = [item.key for item in FormVariable.get_static_data()]
 
         existing_form_key_combinations = []
-        for item in attrs:
+        errors = {}
```

Comment:
probably a `defaultdict(list)` would be more accurate, since you can have multiple errors for the same item/field